smart-contracts/contracts/interfaces/ILock.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity 0.4.24;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "./ERC721.sol";
+import "./ILockCore.sol";
+
+
+/**
+ * @title The Lock interface
+ * @author Julien Genestoux (unlock-protocol.com)
```

Comment:
Feel free to replace these `@author` stuff with your name if you want (but keep `unlock-protocol.com`) or just `unlock-protocol.com`